.github/ISSUE_TEMPLATE/1.bug_report.yaml

```
@@ -65,10 +68,10 @@ body:
       label: What browsers are you seeing the problem on?
       multiple: true
       options:
-        - Firefox
         - Chrome
-        - Safari
+        - Firefox
         - Microsoft Edge
+        - Safari
```

Let's add RN support here too maybe?
```suggestion
        - Safari
        - iOS (React Native)
        - Android (React Native)
```